### DISCUSSION TITLE
搜索功能显示问题

### DISCUSSION THREAD

**Discussion by snxdm** [MaLuns/hexo-theme-async#118] [Q&A 疑难杂症]
部署在cf上后，搜索显示有些问题，在本地不会出现
![Screenshot_2024_0329_233946](https://github.com/MaLuns/hexo-theme-async/assets/151267652/6314078f-a34c-4070-bdce-7ff889c2e552)

**Comment by fujiwaranatsuho** [comment]
我的搜索框也是这样诶，出现在最底部
